pkg/cluster/internal/kubeadm/config.go

```diff
@@ -106,6 +108,8 @@ controllerManagerExtraArgs:
   enable-hostpath-provisioner: "true"
 nodeRegistration:
   criSocket: "/run/containerd/containerd.sock"
+networking:
+  podSubnet: {{ .PodSubnet }}
```

Comment:
I had issues when using IPv6 addresses, we should quote it
`podSubnet: "{{ .PodSubnet }}"`

Reply:
done